Essentials/src/messages.properties

```diff
@@ -573,3 +573,4 @@ msgEnabled=\u00a76Receiving messages \u00a7cenabled\u00a76.
 msgEnabledFor=\u00a76Receiving messages \u00a7cenabled \u00a76for \u00a7c{0}\u00a76.
 msgIgnore=\u00a7c{0} \u00a74has messages disabled.
 minimumPayAmount=\u00a7cThe minimum amount you can pay is {0}.
+commandCooldown=\u00a7cYou cannot type that command for {0}.
```

Comment:
`\u00a7cYou cannot use the {0} command for {1}.` sounds better to me, and provides the command as well.